src/xrCore/xrCore.cpp

```diff
@@ -275,8 +275,9 @@ void xrCore::Initialize(pcstr _ApplicationName, pcstr commandLine, LogCallback c
 #ifdef _EDITOR // for EDITORS - no cache
         flags &= ~CLocatorAPI::flCacheFiles;
 #endif // _EDITOR
+#ifndef DISABLE_PORTABLE_MODE
         flags |= CLocatorAPI::flScanAppRoot;
-
+#endif
```

Comment:
Maybe we can just in runtime if it's system folder?

Reply:
Let's just merge the PR, as is for now. It's ok as is.

Reply:
Maybe we'll find a robust way to determine a system folder.
I was thinking in terms of some sort of a function from SDL2 or the system itself :)